lib/aws-xray-sdk/facets/net_http.rb

```diff
@@ -16,12 +16,18 @@ def new(*options)
     module HTTPInstanceInterceptor
       include XRay::Facets::Helper
 
+      # Pattern matches the path of requests from the AWS Lambda Ruby Runtime.
+      # We do not want to capture these
+      LAMBDA_RUNTIME_PATH_PATTERN = /\A\/[0-9-]{10}\/runtime\/invocation\//.freeze
```

Comment:
Instead of using regex for pattern matching, I think we can fetch the value of `AWS_LAMBDA_RUNTIME_API` environment variable and check if the req.path begins with this value. I just came across this env variable in [lambda runtime](https://docs.aws.amazon.com/lambda/latest/dg/runtimes-api.html) and we can avoid using regex matching altogether.

Reply:
`req.path` would only give the partial path that does not include the host information. Instead `req.uri` gives the complete url `http://127.0.0.1:9001/.....`. Doing a `start_with?` check on `req.uri` would work better. What do you think?

Reply:
Changed to test against the `AWS_LAMBDA_RUNTIME_API` environment variable.
Tested in sample app and seems to be working fine.